api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentServiceTest.java

```diff
@@ -184,21 +184,16 @@ private PaymentCycle createAndSavePaymentCycle() {
     }
 
     private void verifyFailedPaymentSavedWithAllData(PaymentCycle paymentCycle, int amountToPay, String paymentRequestReference, FailureEvent failureEvent) {
-        Payment actualPayment = verifyFailedPaymentSavedCorrectly(paymentCycle, failureEvent);
```

Comment:
github diff isn't very clear here. The payment assertions was unnecessarily being split into two methods so they've been collapsed into one. 
A count of the total payments is now asserted.
No longer checking the id is not null as we already fetched it via the paymentRepository.